ibis/expr/window.py

```diff
@@ -228,10 +274,43 @@ def trailing_window(periods, group_by=None, order_by=None):
     -------
     win : ibis Window
     """
-    return Window(preceding=periods, following=0,
+    return Window(preceding=rows, following=0,
                   group_by=group_by, order_by=order_by)
 
 
+def trailing_time_window(prior_interval, order_by, group_by=None):
+    """
+    Create a trailing time window for use with aggregate window functions.
+
+    Parameters
+    ----------
+    prior_interval : TODO
+    order_by : Union[ir.TimeColumn, ir.DateColumn, ir.TimestampColumn]
+      Specify the column over which the window will look back
+    group_by : expressions, default None
+      Either specify here or with TableExpr.group_by
+
+    Returns
+    -------
+    win: ibis Window
+    """
+
```

Comment:
Still TODO: Be able to pass in prior_interval args such as `5 * day()` and have these args parse to microseconds.